codecov.yml

```diff
@@ -0,0 +1,12 @@
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
```

Comment:
Does this mean it requires 70 - 100% code coverage to pass? Lets discuss tomorrow?

Reply:
Yeah, I want it to fail on the first try. Later we can decrease this number and write more tests.